### DISCUSSION TITLE
Is there a built in way to add a title a LevaPanel?

### DISCUSSION THREAD

**Discussion by alzatin** [pmndrs/leva#484] [Q&A]
Hello, 
I'm pretty new and still trying to figure it all out but right now I have two Leva panels using different stores and I'd like to give them titles that you can see when menus are collapsed. Is there a built-in way to do this?
ty!

**Comment by Jonathan71111** [comment]
Hi there,

Welcome to GitHub! 

Your query about adding titles to the Leva panels, if you're referring to having titles visible when menus are collapsed, it's possible to achieve this using some customization.

One way to accomplish this is by modifying the HTML/CSS of your website. Depending on how the Leva panels are implemented in your site, you might need to inspect the elements or classes associated with these panels.

If the Leva panels are part of your Shopify store, you might want to navigate to the theme files within your Shopify admin panel. From there, you can locate the specific sections or components related to the panels and adjust the code accordingly.

For instance, you can potentially add a title element or modify the existing structure to display titles when the panels are collapsed. However, this might involve some coding knowledge or assistance from a developer familiar with Shopify themes.

To provide more precise guidance, it would be helpful to have a look at the structure of your website. If you're comfortable sharing the link to your site, I can take a closer look and provide more tailored advice on how to achieve this.

One of my client was facing the same issue on [https://pigsnames.com/](https://pigsnames.com/) , which was resolved by the provided methods.
Looking forward to assisting further!

**Reply by alzatin** [reply]
Thank you Jonathan! I was able to achieve what I wanted with the titleBar title prop.
